Participate/chaoss-weekly-and-monthly-calls.md

```diff
@@ -1,7 +1,7 @@
 <a name="calls" id="calls"></a>
 ## CHAOSS Community Calls
 
-Every Tuesday at 18:00 CEST (9am PDT) for one hour. <a href="https://github.com/chaoss/website/releases/download/v1.0/CHAOSS-Calendar_WeeklySync.ics">Add to calendar</a>.
+The CHAOSS community meets weekly on Tuesday at [11 am US Central time](http://arewemeetingyet.com/Chicago/2018-11-06/11:00/w/CHAOSS%20weekly%20meeting#eyJ1cmwiOiJodHRwczovL3Vub21haGEuem9vbS51cy9qLzcyMDQzMTI4OCJ9) for one hour. <a href="https://chaoss.github.io/website/Participate/CHAOSS-Calendar_WeeklySync.ics">Add to calendar (ICS file)</a>.
```

Comment:
I would make this mention some other time zones, to be more welcoming to people in other time zones. Of course, I would add a reference that those times are approximate. For the GMD wg, I used this text:

> We meet every Wednesday at 11am CDT (usually 18:00 CET, but beware different switches to Summer time in EU and US, [check your local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288)